Backend/auction-app/src/main/java/com/example/auctionapp/model/Role.java

```diff
@@ -0,0 +1,54 @@
+package com.example.auctionapp.model;
+
+
+import javax.persistence.*;
```

Comment:
No need to import the whole package just for 3 annotations. You can change intelij wildcard import settings to never squash imports